atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolImpl.java

```diff
@@ -412,8 +412,11 @@ public InetSocketAddress getRandomHostForKey(byte[] key) {
         if (liveOwnerHosts.isEmpty()) {
             log.warn("Perf / cluster stability issue. Token aware query routing has failed because there are no known "
                     + "live hosts that claim ownership of the given range. Falling back to choosing a random live node."
-                    + " Current state logged at DEBUG");
```

Comment:
I also added SafeArg for the current pool state (about 100 lines up from here): https://github.com/palantir/atlasdb/pull/2517/files#diff-a49ee12bdaa4c05d79715ef4cafedd67R327